src/atoms/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,16 @@
+import 'jest-dom/extend-expect';
+import React from 'react';
+// import { fireEvent, render } from 'react-testing-library';
+import { render } from 'react-testing-library';
+
+import { Switch } from 'atoms';
+test('Switch', () => {
+  const handleClick = jest.fn();
+  //   const { getByTestId } = render(
+  //     <Switch data-testid="toggle" handleChange={handleClick} />,
+  //   );
+  //   const Toggle = getByTestId('toggle');/
+  //   fireEvent.click(Toggle);
+  //   expect(handleClick).toHaveBeenCalled();
+  render(<Switch data-testid="toggle" handleChange={handleClick} />);
```

Comment:
Normally the `data-testid` prop would be inside the component's implementation. Alternatively if we set up the label tags to toggle the `Switch` (which is a good idea for accessibility), we can use `getByLabelText` instead of `getByTestId`, without the extra prop. As far the the actual testing logic, I would mainly pass it a `jest.fn()` mock and assert it's called check it's clicked.

I'm also down for a call if you'd like more hands-on help with testing.

Reply:
I'll definitely keep that in mind. Allowing the labels to toggle the `Switch` was an easy change so I've done with that suggestion for this. That makes it simple to test too!

Reply:
Actually it's failing the linter since the labels don't contain the toggle switch itself which is apparently a deprecated rule? I'm interested in your thoughts on this: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-for.md

```
        <LabelText>
          <label htmlFor="toggle">{labelLeft}</label>
        </LabelText>
```

I changed them to look like this